docs/html/reference/pip_install.rst

```diff
@@ -561,8 +561,14 @@ See the :ref:`pip install Examples<pip install Examples>`.
 SSL Certificate Verification
 ----------------------------
 
-Starting with v1.3, pip provides SSL certificate verification over https, to
-prevent man-in-the-middle attacks against PyPI downloads.
+Starting with v1.3, pip provides SSL certificate verification over HTTP, to
+prevent man-in-the-middle attacks against PyPI downloads. This is handled by
+`requests <https://pypi.org/project/requests/>`_ which by default uses a
+bundled CA certificate store provided by
+`certifi <https://pypi.org/project/certifi/>`_ and does not use the system
+certificate store. This may by overridden by using ``--cert`` option or by
+using ``REQUESTS_CA_BUNDLE`` or ``CURL_CA_BUNDLE`` `environment variables <https://requests.readthedocs.io/en/latest/user/advanced/#verification>`_
```

Comment:
Is `PIP_CERT` also supported? It appears in a number of questions online but I can't find support for it in pip source code.

Reply:
I guess it must since in my test `python3 -m pip install requests` failed but `PIP_CERT=/path/to/cert.crt python3 -m pip install requests` worked.

Reply:
pip automatically detects environment variables prefixed by `PIP_*` and read them as flags. Setting `PIP_CERT` has the same effect as passing `--cert`.

Reply:
I don’t think we should mention pip internals here. Since both requests and certifi are vendored as a part of pip, it is sufficient to mention pip bundles a certificate store and uses it by default.

Reply:
@uranusjr simplified, LMK what you think now.